scheds/rust/scx_flash/src/main.rs

```diff
@@ -142,6 +142,15 @@ impl<'a> Scheduler<'a> {
             warn!("vfs_fsync_range symbol is missing")
         }
 
+        // Set scheduler flags.
+        skel.struct_ops.flash_ops_mut().flags = *compat::SCX_OPS_ENQ_EXITING
+            | *compat::SCX_OPS_ENQ_LAST
+            | *compat::SCX_OPS_ENQ_MIGRATION_DISABLED;
+        info!(
```

Comment:
Logging flags is a good idea, maybe we should put that in the helper macro.

Reply:
Yes! Good idea, I have exactly the same code in bpfland.